### DISCUSSION TITLE
Suggestion: Invite to GitHub Discussions on wintercms.com (more than to discord)

### DISCUSSION THREAD

**Discussion by kff5000** [wintercms/winter#76] [Ideas]
Hi,

short introduction of myself: I am not a developer, but I somehow managed to get an october based website up and running a few years ago. And it still is running. So soon I'll have to decide, whether to go the october or winter path. (Most likely winter I think - not because of a few bucks, but because as a non-developer I don't need tickets & premium support, but a vivid, enthusiastic community. Paid, closed source projects almost always lack in this. Already went through this more than a decade ago with Invison Powerboard - things took horrible directions once it got paid - after being "free forever" ;-)

And this is my point: Originally one reason I decided to go with october was the back then busy community support forum. Whenever I had a problem, I could search it or just google - and found a solution in the forum.

But over the past months or maybe years, activity there decreased a lot. Very discouraging for newbies trying to get a grip on the system. Many questions unanswered forever. Why? Because first came october on Slack, then on Discord and lately on GitHub discussions. So where should one ask? On the website forum: no answer. Hmm.. Discord? -> Maybe the question got answered there right away. But buried instantly and invisible to google and therfore everbody of the maybe hundreds who had or have the very same question later. I mean, Stack Overflow isn't just a slack channel for a reason ;-)

I suggest to stick with GitHub Discussions for Support. If someone asks some support question on Discord, ask him to ask for support here. This way question and answer help many, many current and future winter users. Not only this one guy who asked. Also when someone has a question or suggestion, he doesn't have to ask himself, where to ask in the first place - and than maybe even post the same question copy&paste in all 2,3,4 available places to stay on the safe side...

Fist step could be to add a link to GitHub Discussions on wintercms.com. Or even replace the Discord link with it. I strongly believe, that chat just isn't the right medium for community support.

my 2 cents (and sorry for my maybe a bit strange english - I'm from Germany)
best,
Chris

**Comment by LukeTowers** [comment]
Getting close to launching the real website, and we'll be emphasizing the use of GitHub discussions on the Support page :) Thanks for your thoughts on the matter!

**Comment by bennothommo** [comment]
@kff5000 I have posted in our Discord to recommend the use of GitHub Discussions for asking questions, so hopefully we should see some more activity here. :)

**Comment by pablorq** [comment]
From a developer point of view: it's better to have it altogether.

Nr 1: github
Nr 2: stackoverflow

Other channels, like discord or slack, could be nice to interact but hard to find a needle in a haystack.

But the way, nice job!
